tests/test_http_parser.py

```diff
@@ -730,6 +730,21 @@ def test_http_response_parser_no_reason(response: Any) -> None:
     assert msg.reason == ""
 
 
+def test_http_response_parser_lenient_headers(response: Any) -> None:
```

Comment:
Could you replace `Any` with an actual type?

Reply:
I've got an in-progress PR to add more typing to all the tests, so I will update all of them in the file in that PR.

Reply:
Also note the type: ignore on line 1, which means this file is currently not checked at all (I'm going through every file that has been ignored and updating the one at a time).

Reply:
Isn't it easier to start adding typing to new things right away?

Reply:
Figuring out the pytest fixture annotations always takes me a bit of time, and I don't want to duplicate that effort. I also can't test it without introducing 100+ mypy errors. So, in this case, it's easier to do it all at once with a find/replace.

Reply:
marking as resolved, then.